.travis.yml

```diff
@@ -1,7 +1,18 @@
 language: c
-install: wget https://raw.githubusercontent.com/ocaml/ocaml-travisci-skeleton/master/.travis-opam.sh
-script: bash -ex .travis-opam.sh
-dist: trusty
+sudo: false
+services:
+  - docker
+install: wget https://raw.githubusercontent.com/ocaml/ocaml-travisci-skeleton/master/.travis-docker.sh
+script: bash ./.travis-docker.sh
 env:
-  - PACKAGE="xenctrl" OCAML_VERSION=4.01
-  - PACKAGE="xenctrl" OCAML_VERSION=latest
+ global:
+   - PACKAGE="xenctrl"
+ matrix:
+   - DISTRO=debian-stable OCAML_VERSION=4.01.0
+   - DISTRO=debian-testing OCAML_VERSION=4.02.3
+   - DISTRO=debian-unstable OCAML_VERSION=4.03.0
+   - DISTRO=ubuntu-12.04 OCAML_VERSION=4.02.3
```

Comment:
Is there a backport available? I can get those installed in the 12.04 container and document it.  Alternatively I can just comment it out.  Same question for CentOS 7 as well -- is there an RPM remote available I should add?

Looks like Alpine fails for a different reason:
```
- mktemp: Invalid argument
- BusyBox v1.24.2 (2016-08-12 14:38:34 GMT) multi-call binary.
- 
```